rfcs/0087-aarch64-tier1.md

```diff
@@ -51,6 +51,20 @@ to block channel advances. Hydra will start building aarch64 packages and run
 aarch64-based tests as part of stable and unstable channels, including them in
 the binary cache, increasing its coverage as a result.
 
+A team for aarch64-specific build failures should be established (or, more
+precisely, revived, as per [RFC0046](https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md) there should already be a team for ARM platforms called @NixOS/aarch64-maintainers)
+to help track down and fix breaking changes specific to ARM platforms and be
+at the front line of battle for ARM stability.
+
+Additionally, maintainers of critical packages (e.g. binutils) should be given
+an advance notice before implementing the change to ensure no delays in channel
+advances happen and no surprises occur, such as suddenly increased workload. An
+option would be to ping them on the PR implementing this RFC and discuss it there
+(and potentially delay the merge if there is any pushback).
```

Comment:
Thanks. This looks like a concise & accurate summary of our offline email discussion from 16 August 2021, IMO.

Reply:
I tried to summarize it as best as possible while making it actionable for implementers by suggesting an option that felt the most appropriate. Thank you.